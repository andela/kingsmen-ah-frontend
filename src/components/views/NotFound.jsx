import React from 'react';

const NotFound = () => (
  <div>
    <h1>This page is not found</h1>
  </div>
);

export default NotFound;
