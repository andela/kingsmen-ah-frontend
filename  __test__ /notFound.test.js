import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import NotFound from '@components/views/NotFound';

configure({ adapter: new Adapter() });

const setup = () => {
  const wrapper = shallow(<NotFound />);
  return wrapper;
};

describe('<NotFound />', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = setup();
  });

  it('should display this page is not found', () => {
    const text = wrapper.find('h1').text();
    expect(text).toEqual('This page is not found');
  });

  it('should render the NotFound component correctly', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
