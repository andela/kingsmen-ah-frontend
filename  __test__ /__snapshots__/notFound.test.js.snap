// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should render the NotFound component correctly 1`] = `
<div>
  <h1>
    This page is not found
  </h1>
</div>
`;
